### PR TITLE
COM-29 Apply logic to lab results form

### PR DESCRIPTION
**Pull request by lauravignoli**

### CODE REVIEW

**Comment by lauravignoli** [comment]
I am getting an error sayiing: "No element found with name : CSF parasitology Test Date" but can't find the origin of it and it works fine. Might be due to caching maybe.

File: openmrs/apps/clinical/formConditions.js [hunk 1]
```diff
@@ -382,5 +382,1566 @@ Bahmni.ConceptSet.FormConditions.rules = {
...
+		if (value && value !== "") {	
+			conditions.show.push("Vaginal Cervical smear Yeast cells per field Test Date");
+		} else {	
+			conditions.hide.push("Vaginal Cervical smear Yeast cells per field Test Date");
+		}	
+		return conditions;	
+	},
+	'Vaginal Cervical smear Epithelial cells per field': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};		
+		var value = formFieldValues['Vaginal Cervical smear Leucocytes per field'];
```
Comment (tmvumbi2) [inline openmrs/apps/clinical/formConditions.js]: @lauravignoli Did you mean `Vaginal Cervical smear Epithelial cells per field` here?

File: openmrs/apps/clinical/formConditions.js [hunk 2]
```diff
@@ -382,5 +382,1566 @@ Bahmni.ConceptSet.FormConditions.rules = {
...
+		if (value && value !== "") {	
+			conditions.show.push("Urethral smear Epithelial cells Test Date");
+		} else {	
+			conditions.hide.push("Urethral smear Epithelial cells Test Date");
+		}	
+		return conditions;
+	},
+	'Urethral smear Trichomonas vaginalis': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Urethral smear Leucocytes (WBC)'];
```
Comment (tmvumbi2) [inline openmrs/apps/clinical/formConditions.js]: @lauravignoli Did u mean `Urethral smear Trichomonas vaginalis` here?

File: openmrs/apps/clinical/formConditions.js [hunk 3]
```diff
@@ -382,5 +382,1566 @@ Bahmni.ConceptSet.FormConditions.rules = {
...
+		if (value && value !== "") {	
+			conditions.show.push("Total cholesterol Test Date");
+		} else {	
+			conditions.hide.push("Total cholesterol Test Date");
+		}	
+		return conditions;	
+	},
+	'HDL': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['HDL '];
```
Comment (tmvumbi2) [inline openmrs/apps/clinical/formConditions.js]: @lauravignoli please remove the space after `HDL`

Comment (lauravignoli) [reply openmrs/apps/clinical/formConditions.js]: Fixed

File: openmrs/apps/clinical/formConditions.js (hunk 2 above)
Comment (lauravignoli) [reply openmrs/apps/clinical/formConditions.js]: Fixed

File: openmrs/apps/clinical/formConditions.js (hunk 1 above)
Comment (lauravignoli) [reply openmrs/apps/clinical/formConditions.js]: Fixed

**Pull request merged**